wallet/src/test/java/io/neow3j/wallet/Nep5Test.java

```diff
@@ -0,0 +1,52 @@
+package io.neow3j.wallet;
+
+import io.neow3j.contract.ScriptHash;
+import io.neow3j.protocol.Neow3j;
+import io.neow3j.protocol.http.HttpService;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.math.BigInteger;
+
+public class Nep5Test {
+
+    private Neow3j neow3j;
+    private ScriptHash contractScriptHash;
+
+    @Before
+    public void setUp() {
+        this.neow3j = Neow3j.build(new HttpService("http://localhost:30333"));
```

Comment:
Currently, the tests are failing on Travis. This is because you're using "localhost:30333" as the Neo node.
You **must** mock the neow3j behavior to only test the Nep5 functionality, and not other code abstraction layers.
Please check the class `ContractDeploymentTest.class` for an example on how to mock and test higher layers that rely on the `Neow3j` class.